server/@types/approved-premises/index.d.ts

```diff
@@ -9,6 +9,7 @@ declare module 'approved-premises' {
   export type Cancellation = schemas['Cancellation']
   export type BookingExtension = schemas['BookingExtension']
   export type KeyWorker = schemas['KeyWorker']
+  export type DateCapacity = schemas['DateCapacity']
```

Comment:
I think this might be better expressed as `PremisesCapacityItem` and then `PremisesCapacity` can be an `Array<PremisesCapacityItem>`